### DISCUSSION TITLE
开启了网站浏览统计似乎有时候会导致当前浏览页面刷新

### DISCUSSION THREAD

**Discussion by black-zero358** [Planshit/Tai#180] [General]
不知道有没有人遇到了类似的问题，概率比较低，也不确实是不是因为开启了网站浏览统计功能导致的

**Comment by Wenbobobo** [comment]
我是开启后100%浏览器崩溃，关了就好了（

**Reply by noberumotto** [reply]
请提供一下浏览器名称和版本号

**Reply by Wenbobobo** [reply]
Microsoft Edge
版本 113.0.1774.50 (正式版本) (64 位)
Tai版本是最新1.5.0.1（1.5.0开始使用就发现此问题）
卡死时内存占用约80%，Edge占约2.2G/16G，其他程序完全正常
具体表现是打开该选项后点击浏览器即为无响应状态，关闭选项后强制重启浏览器就正常了

**Reply by Wenbobobo** [reply]
> Microsoft Edge 版本 113.0.1774.50 (正式版本) (64 位) Tai版本是最新1.5.0.1（1.5.0开始使用就发现此问题） 卡死时内存占用约80%，Edge占约2.2G/16G，其他程序完全正常 具体表现是打开该选项后点击浏览器即为无响应状态，关闭选项后强制重启浏览器就正常了

![image](https://github.com/Planshit/Tai/assets/78262508/73303ec5-cf58-41b4-8f5e-269ecfc50899)
最近一次测试时状态，展开约20标签页，**大部分处于休眠中**，含标签页组约120标签页

**Comment by noberumotto** [comment]
观察一下电脑内存状态，内存不够的情况下会导致浏览器页面重新加载的

**Reply by black-zero358** [reply]
16g的运存，浏览器重载时倒没注意过内存情况，但确实在只打开较少页面的情况下很少发生重载的情况
